src/relay/op/tensor/reduce.cc

```diff
@@ -207,9 +208,29 @@ Array<te::Tensor> ReduceCompute(const Attrs& attrs, const Array<te::Tensor>& inp
       return {topi::identity(inputs[0])};
     }
   }
+
   return {f(inputs[0], axes, param->keepdims, false)};
 }
 
+template <typename F>
+Array<te::Tensor> OneElementReduceCompute(const Attrs& attrs, const Array<te::Tensor>& inputs,
```

Comment:
I don't understand what OneElement means here?

Reply:
Hmm, I've changed it to ArgReduceCompute to be consistent with the other naming suggested by Masa.